.github/workflows/build-and-cache.yaml

```diff
@@ -28,16 +28,23 @@ jobs:
   build:
     strategy:
       matrix:
-        os: [ubuntu-latest, macos-latest, macos-13]
+        os: [ ubuntu-latest, macos-latest, macos-13 ]
```

Comment:
New formatter? ;-)

Reply:
Just my IDE

Reply:
Cool, is there a formatter we can use? Doesn't need to be on this PR.